version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.31.0"
+version in ThisBuild := "1.31.1"
```

Comment:
I'll handle the versions myself :wink: I've already done enough incompatible changes that `1.32.0` will be the next one.